src/applications/check-in/api/versions/v2.js

```diff
@@ -74,10 +74,11 @@ const v2 = {
     };
   },
 
-  getCheckInData: async token => {
+  getCheckInData: async (token, reload = false) => {
     const url = '/check_in/v2/patient_check_ins/';
+    const typeSlug = reload ? '?reload=true' : '?reload=false';
```

Comment:
NITPICK: For future proofing, maybe use some URL param builder instead of creating it yourself

example: https://github.com/department-of-veterans-affairs/vets-website/pull/20673/files#diff-db47f71922a221fa2b48ae702ce3cdd34e6f0d5cb6b1b5f3fe275049ed10d409R42

Reply:
oh cool, I'll check that out.